mathesar_ui/src/systems/table-view/table-inspector/column/SetDefaultValue.svelte

```diff
@@ -17,7 +19,13 @@
 
   $: initialValue = column.column.default?.value ?? column.initialInputValue;
   $: value = initialValue;
+  $: isDefaultNull = column.column.default === null;
   $: actionButtonsVisible = (() => {
+    if (isDefaultNull) {
+      if (typeof value === 'object') {
+        JSON.stringify(initialValue) !== '';
+      } else return String(initialValue) !== '';
```

Comment:
A simple check of the form: 

```typescript

if (isDefaultNull) {
    return column.column.default !== null;
}

```
should work just fine.

Reply:
Yes, it's working fine.